CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file, per [the Ke
 
 ## [Unreleased] - TBD
 
+## [1.3.3] - 2024-06-26
```

Comment:
@jeffpaul do you mind adding the link we'll need at the bottom of the file? 

```
[1.3.3]: https://github.com/10up/wp-newrelic/compare/1.3.2...1.3.3
```

Reply:
Good catch, fixed in fa48f39.